src/utils/file.js

```diff
@@ -183,6 +183,8 @@ export async function createFile(fileName, content, debug = false) {
   }
 
   return new Promise(async (resolve, reject) => {
+    fileName = fileName.split('/').join(path.sep)
```

Comment:
Definitely worth adding a test or two for this.

Reply:
Moved this logic to the `unifyFilePath` utility and added unit test for it.